src/Libraries/CoreNodeModels/CoreNodeModels.csproj

```diff
@@ -48,9 +48,11 @@
     <Reference Include="System" />
     <Reference Include="System.Core" />
     <Reference Include="System.Web" />
+    <Reference Include="System.Xaml" />
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
+    <Compile Include="Input\ColorPalette.cs" />
```

Comment:
ahh, I see you have the file above in the PR but its not in the compile includes... just delete it I think right? Seems to be included by accident.